README.md

```diff
@@ -68,17 +68,38 @@ swag init -g cmd/main.go
 
 ## Frontend dev server
 
-In order to install/update packages in the node-urdr server you should:
+In order to run the node-urdr server in a Docker container, you don't need to install frontend packages locally. However, for your IDE (e.g. VSCode) to deal properly with React and other packages, a local install might be needed. To do that:
 
-- Make the neccessary updates in the package.json
-- Execute the following commands:
+- Move to the `/frontend` directory
+- Run `npm ci`.
+
+### Installing and updating packages
+
+In order to minimize dependency differences between local environments we create our `package-lock.json` file in a separate Docker container.
```

Comment:
In order to avoid dependency differences...By doing this you remove the possibility of having differences.